client/my-sites/plans/test/index.js

```diff
@@ -18,7 +18,7 @@ jest.mock( 'my-sites/controller', () => ( {
 	siteSelection: jest.fn(),
 	sites: jest.fn(),
 } ) );
-jest.mock( 'lib/abtest/getters', () => ( {
+jest.mock( 'lib/plans/config', () => ( {
```

Comment:
Noting that I'd prefer to completely get rid of these tests, as I don't really see the value in having them.

Reply:
I think they had value during development when we needed to be absolutely sure that only users in the experiment were seeing the new plans page but now they don't serve any purpose.

Once we know for sure we are not reverting anything back, we will remove the `shouldShowOfferResetFlow` check from everywhere and remove these tests as well. How that sounds to you?